Tasks/AzureFileCopyV2/task.json

```diff
@@ -13,8 +13,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 154,
-        "Patch": 19
+        "Minor": 166,
```

Comment:
Should we make this change to the latest task version?

Reply:
Does this fix cover all versions? We're using V4 now and still have the same issue. We used V3 previously. We have never used V2.